### PR TITLE
SGA-11041 - fixes provider on empty strings in different lists

### PR DESCRIPTION
**Pull request by otlg**
- dataset-owner
- security_policy_id
- include_locations

### CODE REVIEW

File: satori/resources/resource_common.go [hunk 1]
```diff
@@ -123,17 +125,21 @@ func setMapProp(in *map[string]interface{}, prop string, d *schema.ResourceData)
 	return nil
 }
 
-func getStringListProp(prop string, d *schema.ResourceData) *[]string {
+func getStringListProp(prop string, d *schema.ResourceData) (*[]string, error) {
 	if raw, ok := d.GetOk(prop); ok {
 		in := raw.([]interface{})
 		list := make([]string, len(in))
 		for i, v := range in {
+			log.Printf("getStringListProp, v=: %s", v)
```
Comment (shalomtuby) [inline satori/resources/resource_common.go:133]: remove logs

File: satori/resources/resource_taxonomy_classifier.go [hunk 2]
```diff
@@ -186,7 +186,8 @@ func resourceToClassifier(d *schema.ResourceData) (*api.TaxonomyClassifier, erro
 		}
 		if _, ok := d.GetOk("custom_config.0.values"); ok {
 			var values api.TaxonomyClassifierValues
-			values.Values = getStringListProp("custom_config.0.values", d)
+			valuesConfig, _ := getStringListProp("custom_config.0.values", d)
```
Comment (shalomtuby) [inline satori/resources/resource_taxonomy_classifier.go:189]: Why do you ignore the error?

**Review by shalomtuby — APPROVED** [review]
see comments

File: satori/resources/resource_taxonomy_classifier.go (hunk 2 above)
Comment (otlg) [reply satori/resources/resource_taxonomy_classifier.go:189]: Not interested for now

File: satori/resources/resource_common.go (hunk 1 above)
Comment (otlg) [reply satori/resources/resource_common.go:133]: I want these logs :)

**Pull request merged**